src/modules/drawing/components/DrawingTooltip.vue

```diff
@@ -86,28 +86,29 @@ function onPointerMove(event) {
     let featureDrawingMode = featureUnderCursor?.get('type').toUpperCase()
     let translationKeys
 
-    if (hoveringSelectedFeature) {
-        mapElement.classList.add(cssGrab)
-        mapElement.classList.remove(cssPointer)
-
+    if (hoveringSelectedFeature && !drawingMode.value) {
```

Comment:
Could you try changing a bit the order of if/else if/if ? I feel like having `if (drawingMode.value)` first, then `else if(hoveringSelectedFeature)` and finally `else if (hoveringSelectableFeature)`, would clarify a bit more what's happening. (but there's a little `else` block to take care of, don't know if that's possible)